en/Formulaic Privacy Notice

```diff
@@ -0,0 +1,77 @@
+# Formulaic Privacy Notice
+
+Last Updated March 8, 2024
+{: datetime="2024-03-08" }
+
+This privacy notice applies to Formulaic, a service that allows users to query, create, share, and manage templates for querying artificial intelligence (AI) based large language models (LLMs) (the “Service”). The Service is owned, licensed, and operated by Mozilla Corporation (referred to herein as "we," "us," "our," or "Mozilla"). 
```

Comment:
Use curly quotes (not straight quotes) as in `“Service”` for the other ones in this paragraph